### PR TITLE
Added error handle for division forms

### PR DESCRIPTION
**Pull request by sakthivel-axim**
## Status
**READY**

## Description
Added error handling to forms division.

https://trello.com/c/D7L2SUrL

### CODE REVIEW

File: lib/dynamic_screening_solutions/api.py [hunk 1]
```diff
@@ -333,6 +333,8 @@ def get_forms_by_division(self, division_id):
         request_url = self.get_request_url(resource_path=resource_path)
         response = self.request_get(request_url)
         forms = response.json()
+        if 'error' in forms and forms['error'] is True:
+            forms = []
```
Comment (jontsai) [inline lib/dynamic_screening_solutions/api.py:337]:
```suggestion
        try:
            forms = response.json()
            if type(forms) == 'dict' and forms.get('error'):
                forms = []
      except Exception:
          forms = None
```

**Review by jontsai — CHANGES_REQUESTED** [review]

File: lib/dynamic_screening_solutions/api.py (hunk 1 above)
Comment (sakthivel-axim) [reply lib/dynamic_screening_solutions/api.py]: Done

Comment (sakthivel-axim) [reply lib/dynamic_screening_solutions/api.py]: @jontsai the try/except is not added for many functions in the file `lib/dynamic_screening_solutions/api.py`.
Should I add a similar try/except handle for other functions as well?

**Comment by jontsai** [comment]
Closing this in favor of https://github.com/hacktoolkit/django-htk/commit/70c535de69bc6158055c9e846ac1c9190b9fd0dd

**Pull request closed without merging**